src/cascadia/TerminalApp/ActionArgs.h

```diff
@@ -531,6 +532,61 @@ namespace winrt::TerminalApp::implementation
             return { *args, {} };
         }
     };
+
+    
+    // Possible AnchorKey values
+    static constexpr std::string_view ControlString{ "ctrl" };
+    static constexpr std::string_view AltString{ "alt" };
+    static constexpr std::string_view ShiftString{ "shift" };
+
+    static TerminalApp::AnchorKey ParseAnchorKey(const std::string& anchorString)
+    {
+        if (anchorString == ControlString)
+        {
+            return TerminalApp::AnchorKey::Ctrl;
+        }
+        else if (anchorString == AltString)
+        {
+            return TerminalApp::AnchorKey::Alt;
+        }
+        else if (anchorString == ShiftString)
+        {
+            return TerminalApp::AnchorKey::Shift;
+        }
+        // default behavior for invalid data
+        return TerminalApp::AnchorKey::None;
+    };

```

Comment:
Oh yeah they're really neat! I'm going to punt this for later though because I believe I'll need to wait for a couple of changes to JsonUtilsNew.h before I can migrate over.

Reply:
migrated over, it's great stuff 👍